defaults/main.yml

```diff
@@ -28,28 +38,66 @@ php_scl_install_php: yes
 php_scl_php_packages:
   - "{{ php_scl_name }}"
   - "{{ php_scl_name }}-php-cli"
+  - "{{ php_scl_name }}-php-pdo"
+  - "{{ php_scl_name }}-php-mysql"
+  - "{{ php_scl_name }}-php-gd"
 
 # Should this role manage the configuration of PHP?
 #
 # yes | no
 php_scl_configure: yes
 
 # Timezone to set in the php.ini file
-php_scl_timezone: UTC
+php_scl_timezone: America/New_York
 
 # List of configuration files and options to insert
-#
+
+php_realpath_cache_size: 512k
+php_realpath_cache_ttl: 3600
+php_max_execution_time: 60
+php_max_input_vars: 2000
+php_memory_limit: 512M
+php_error_reporting: E_ALL & ~E_DEPRECATED
+php_html_errors: Off
+php_post_max_size: 10M
+php_upload_max_filesize: 10M
+php_default_socket_timeout: 300
+php_mysql_connect_timeout: 300
+php_upload_tmp_dir: "/var/tmp"
 # Will only be added if `php_scl_configure is `yes`|`true`
 #
 # - file: /absolute/file/path.ini
 #   values:
 #     - { section: ini_section, key: config_key, value: config_value }
 #     - { section: ini_section, key: config_key2, value: another_value }
 php_scl_configure_options:
-  - file: "/opt/rh/{{ php_scl_name }}/root/etc/php.ini"
+  - file: "/etc/opt/rh/{{ php_scl_name }}/php.ini"
     values:
+      - { section: PHP, key: realpath_cache_size, value: "{{ php_realpath_cache_size }}" }
+      - { section: PHP, key: realpath_cache_ttl, value: "{{ php_realpath_cache_ttl }}" }
+      - { section: PHP, key: max_execution_time, value: "{{ php_max_execution_time }}" }
+      - { section: PHP, key: max_input_vars, value: "{{ php_max_input_vars }}" }
+      - { section: PHP, key: memory_limit, value: "{{ php_memory_limit }}" }
+      - { section: PHP, key: error_reporting, value: "{{ php_error_reporting }}" }
+      - { section: PHP, key: html_errors, value: "{{ php_html_errors }}" }
+      - { section: PHP, key: post_max_size, value: "{{ php_post_max_size }}" }
+      - { section: PHP, key: upload_max_filesize, value: "{{ php_upload_max_filesize }}" }
+      - { section: PHP, key: upload_tmp_dir, value: "{{ php_upload_tmp_dir }}" }
+      - { section: PHP, key: default_socket_timeout, value: "{{ php_default_socket_timeout }}" }
+      - { section: MySQL, key: mysql.connect_timeout, value: "{{ php_mysql_connect_timeout }}" }
```

Comment:
I think all of this can be done in the end project's inventory, does not need to be done in the role.

Reply:
Yea, actually that make sense to keep them default in there.